cli/tests/test_util_command.rs

```diff
@@ -112,3 +112,13 @@ fn test_shell_completions() {
     test("nushell");
     test("zsh");
 }
+
+#[cfg(unix)]
```

Comment:
Nit: Cross-platform tests, since we cannot just implement the feature purely for Unix.

Reply:
Do you have an idea for a command that's always available on Windows? I don't use Windows and I don't want to debug with a CI that takes over 5mins to run.

Reply:
Use test tools introduced by `jj fix`.